src/platformAccessory.ts

```diff
@@ -39,14 +39,58 @@ export class SalusSQ610Accessory {
       maxValue: 40,
       minStep: 0.5,
     }).onSet(this.onTargetTemperatureSet.bind(this));
+    this.service.getCharacteristic(this.platform.Characteristic.TargetHeatingCoolingState).setProps({
+      validValues: [this.platform.Characteristic.TargetHeatingCoolingState.OFF,
+        this.platform.Characteristic.TargetHeatingCoolingState.HEAT],
```

Comment:
Salus thermostats can operate also in "COOLING" mode by shorting the pin in the back or manually setting it up.